--- ming-0.3.0/perl_ext/Makefile.PL.orig	2006-02-01 07:45:17.000000000 +0100
+++ ming-0.3.0/perl_ext/Makefile.PL	2007-02-16 23:14:25.929759689 +0100
@@ -78,7 +78,7 @@
     clean   => {
         FILES   => "@do_clean",
     },
-    'MYEXTLIB'     => $MING_DIR . '/libming.a',
+    'MYEXTLIB'     => $MING_DIR . '/libming.so',
     'OBJECT'       => "$XS_OBJECTS",
     #'LIBS'         => [ "-lz" ],
 		'LIBS'         => [ $libs ],
@@ -96,7 +96,7 @@
 }
 
 sub ming_is_compiled{
-	return undef unless (-e "$MING_DIR/libming.a");
+	return undef unless (-e "$MING_DIR/libming.so");
 	return 1;
 }
 
