--- ming-0.4.2/perl_ext/Makefile.PL.orig	2008-07-13 22:14:52.000000000 +0200
+++ ming-0.4.2/perl_ext/Makefile.PL	2008-10-14 22:51:17.854630631 +0200
@@ -146,7 +146,7 @@
     clean   => {
         FILES   => "@do_clean",
     },
-    'MYEXTLIB'     => $MING_TOP_BUILDDIR . '/src/.libs/libming.a',
+    'MYEXTLIB'     => $MING_TOP_BUILDDIR . '/src/.libs/libming.so',
     'OBJECT'       => "$XS_OBJECTS",
     'LDFROM'       => "$LD_OBJECTS",
     'INC'          => "-I$MING_TOP_SRCDIR/src -I$BUILDDIR -I$MING_TOP_BUILDDIR/src -I$SRCDIR",
@@ -164,7 +164,7 @@
 
 sub ming_is_compiled
 {
-	my $libming = $MING_TOP_BUILDDIR."/src/.libs/libming.a";
+	my $libming = $MING_TOP_BUILDDIR."/src/.libs/libming.so";
 
 	if ( -e $libming )
 	{
--- ming-0.4.2/perl_ext/Makefile.am.orig	2008-07-13 15:46:56.000000000 +0200
+++ ming-0.4.2/perl_ext/Makefile.am	2008-10-15 17:58:44.486630631 +0200
@@ -32,13 +32,13 @@
 #abs_builddir=`cd $(TMP_DIR); pwd`
 
 all: Makefile.perl
-	$(MAKE) -f Makefile.perl
+	$(MAKE) -f Makefile.perl OPTIMIZE="$(CFLAGS)"
 
 install: Makefile.perl
-	$(MAKE) -f Makefile.perl install PREFIX=$(prefix)
+	$(MAKE) -f Makefile.perl pure_install PREFIX=$(prefix)
 
 Makefile.perl: Makefile.PL
-	srcdir=$(srcdir) builddir=$(abs_builddir) $(PERL) $< PREFIX=$(prefix) FIRST_MAKEFILE=Makefile.perl
+	srcdir=$(srcdir) builddir=$(abs_builddir) $(PERL) $< PREFIX=$(prefix) FIRST_MAKEFILE=Makefile.perl INSTALLDIRS=vendor
 
 clean-local: Makefile.perl
 	$(MAKE) -f Makefile.perl realclean
