--- ming-0.4.3/util/Makefile.am.orig	2008-12-22 14:33:59.000000000 +0100
+++ ming-0.4.3/util/Makefile.am	2010-03-04 14:32:24.170198782 +0100
@@ -114,16 +114,16 @@
 makefdb_CFLAGS = -DMAKE_FDB
 
 gif2mask_SOURCES = gif2mask.c
-gif2mask_LDADD = $(top_builddir)/src/libming.la
+gif2mask_LDADD = $(GIFLIB) libutil.la $(top_builddir)/src/libming.la
 
 png2dbl_SOURCES = png2dbl.c
-png2dbl_LDADD = $(top_builddir)/src/libming.la
+png2dbl_LDADD = $(PNGLIB) libutil.la $(top_builddir)/src/libming.la
 
 dbl2png_SOURCES = dbl2png.c
-dbl2png_LDADD = $(top_builddir)/src/libming.la
+dbl2png_LDADD = $(PNGLIB) libutil.la $(top_builddir)/src/libming.la
 
 gif2dbl_SOURCES = gif2dbl.c
-gif2dbl_LDADD = $(top_builddir)/src/libming.la
+gif2dbl_LDADD = $(GIFLIB) libutil.la $(top_builddir)/src/libming.la
 
 makeswf_SOURCES = makeswf.c makeswf_utils.c
 makeswf_LDADD = $(top_builddir)/src/libming.la
--- ming-0.4.3/util/Makefile.in.orig	2010-02-06 22:53:35.000000000 +0100
+++ ming-0.4.3/util/Makefile.in	2010-03-04 14:42:29.393680101 +0100
@@ -395,13 +395,13 @@
 makefdb_LDADD = libutil.la $(top_builddir)/src/libming.la
 makefdb_CFLAGS = -DMAKE_FDB
 gif2mask_SOURCES = gif2mask.c
-gif2mask_LDADD = $(top_builddir)/src/libming.la
+gif2mask_LDADD = $(GIFLIB) libutil.la $(top_builddir)/src/libming.la
 png2dbl_SOURCES = png2dbl.c
-png2dbl_LDADD = $(top_builddir)/src/libming.la
+png2dbl_LDADD = $(PNGLIB) libutil.la $(top_builddir)/src/libming.la
 dbl2png_SOURCES = dbl2png.c
-dbl2png_LDADD = $(top_builddir)/src/libming.la
+dbl2png_LDADD = $(PNGLIB) libutil.la $(top_builddir)/src/libming.la
 gif2dbl_SOURCES = gif2dbl.c
-gif2dbl_LDADD = $(top_builddir)/src/libming.la
+gif2dbl_LDADD = $(GIFLIB) libutil.la $(top_builddir)/src/libming.la
 makeswf_SOURCES = makeswf.c makeswf_utils.c
 makeswf_LDADD = $(top_builddir)/src/libming.la
 bin_SCRIPTS = ming-config
