diff -Nur ming-0.2a.orig/py_ext/Makefile ming-0.2a/py_ext/Makefile
--- ming-0.2a.orig/py_ext/Makefile	Fri Aug 17 19:05:55 2001
+++ ming-0.2a/py_ext/Makefile	Sat Oct 12 18:38:05 2002
@@ -4,10 +4,10 @@
 #PYINCDIR = /usr/include/python1.5
 #PYLIBDIR = /usr/lib/python1.5
 
-all:
+all: mingcmodule.so
 
 mingcmodule.so: ming_wrap.o
-	gcc -g -Wall -shared -o mingcmodule.so ming_wrap.o -lming
+	gcc -g -Wall -shared -o mingcmodule.so ming_wrap.o -L.. -lming
 
 mingc.pyd: ming_wrap.o
 	dllwrap --dllname mingc.pyd --driver-name gcc --def mingc.def \
