--- libming-ming-0_4_8/configure.in.orig	2023-07-07 10:32:34.180798498 +0200
+++ libming-ming-0_4_8/configure.in	2023-07-07 10:33:17.083250494 +0200
@@ -217,6 +217,8 @@
 AC_SUBST(TCL_LIB_DIR)
 AC_SUBST(TCL_INC_DIR)
 
+AC_PATH_PROG(SWIG, swig)
+
 dnl --------------------------------------------
 dnl Check for required programs
 dnl --------------------------------------------
--- libming-ming-0_4_8/py_ext/Makefile.am.orig	2017-04-07 10:12:56.000000000 +0200
+++ libming-ming-0_4_8/py_ext/Makefile.am	2023-07-07 10:39:32.337887744 +0200
@@ -12,7 +12,7 @@
 	CFLAGS="$(CFLAGS) -fno-strict-aliasing" $(PYTHON) setup.py build
 
 ming_wrap.c: ../ming.i
-	swig -I$(srcdir)/.. -python ming.i
+	$(SWIG) -I$(srcdir)/.. -outcurrentdir -python $(srcdir)/../ming.i
 
 clean-local:
 	rm -Rf test.swf build
--- libming-ming-0_4_8/tcl_ext/Makefile.am.orig	2017-04-07 10:12:56.000000000 +0200
+++ libming-ming-0_4_8/tcl_ext/Makefile.am	2023-07-07 10:39:26.271372423 +0200
@@ -17,4 +17,4 @@
 EXTRA_DIST = test.tcl MANIFEST
 
 ming_wrap.c: ../ming.i
-	swig -I$(srcdir)/.. -tcl ming.i
+	$(SWIG) -I$(srcdir)/.. -outcurrentdir -tcl $(srcdir)/../ming.i
--- libming-ming-0_4_8/ming.i.orig	2017-04-07 10:12:56.000000000 +0200
+++ libming-ming-0_4_8/ming.i	2023-11-11 21:08:14.719238734 +0100
@@ -11,7 +11,8 @@
 
 %}
 
-%typemap(python,in) FILE * {
+#ifdef SWIGpython
+%typemap(in) FILE * {
   if (!PyFile_Check($input)) {
     PyErr_SetString(PyExc_TypeError, "Need a file!");
     return NULL;
@@ -19,25 +20,29 @@
   $1 = PyFile_AsFile($input);
 }
 
-%typemap(python,in) char * {
+%typemap(in) char * {
   if (!PyString_Check($input)) {
     PyErr_SetString(PyExc_TypeError, "not a string");
     return NULL;
   }
   $1 = PyString_AsString($input);
 }
+#endif
 
 typedef unsigned char byte;
 
-%typemap(python,in) byte * {
+#ifdef SWIGpython
+%typemap(in) byte * {
   if (!PyString_Check($input)) {
     PyErr_SetString(PyExc_TypeError, "not a string");
     return NULL;
   }
   $1 = PyString_AsString($input);
 }
+#endif
 
-%typemap(ruby,in) int * {
+#ifdef SWIGruby
+%typemap(in) int * {
   if (NIL_P($input)) {
          $1 = 0;
   } else {
@@ -45,6 +50,7 @@ typedef unsigned char byte;
     $1 = (int *)RARRAY($input)->ptr;
   }
 }
+#endif
 
 %pointer_functions(int, intp);
 %pointer_functions(float, floatp);
